tools/loader/ExecutorCore.cpp

```diff
@@ -222,6 +222,13 @@ int Executor::executeNetwork(int argc, char **argv) {
       << "singleBatchRepeatedMode is not compatible with "
          "streamInputFilenamesMode";
 
+  // When the mini-batch mode is enabled do not allow debug instrumentation.
+  extern llvm::cl::opt<bool> instrumentDebug;
```

Comment:
This is a bit unusual place for an `extern` declaration. Also, we have some CommandLine.h files for many libs/Backends/. Maybe we should have the same thing for the Optimizer, which would allow us to include such a file instead of sprinkling `extern` for different flags here and there.

Reply:
Created `CommandLine.h` and `ComamndLine.cpp` files. Done.